packages/loot-core/src/client/query-helpers.ts

```
@@ -1,244 +1,332 @@
-export class LiveQuery<Response = unknown> {
-  _unsubscribe;
-  data;
-  dependencies;
-  error;
-  listeners;
-  mappedData;
-  mapper;
-  onlySync;
-  query;
+export class LiveQuery<TResponse = unknown> {
+  private _unsubscribeSyncEvent: () => void | null;
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)